src/dll/Hooks/ExecuteProcess.cpp

```diff
@@ -0,0 +1,100 @@
+#include "ExecuteProcess.hpp"
+#include "Addresses.hpp"
+#include "App.hpp"
+#include "Hook.hpp"
+#include "Systems/ScriptSystem.hpp"
+#include "stdafx.hpp"
+
+namespace
+{
+bool isAttached = false;
+
+struct FixedWString
+{
+    uint32_t length;
+    uint32_t maxLength;
+    wchar_t* str;
+};
+
+bool _Global_ExecuteProcess(void* a1, RED4ext::CString* aCommand, FixedWString* aArgs,
+                            RED4ext::CString* aCurrentDirectory, char a5);
+Hook<decltype(&_Global_ExecuteProcess)> Global_ExecuteProcess(Addresses::Global_ExecuteProcess,
+                                                              &_Global_ExecuteProcess);
+
+bool _Global_ExecuteProcess(void* a1, RED4ext::CString* aCommand, FixedWString* aArgs,
+                            RED4ext::CString* aCurrentDirectory, char a5)
+{
+    if (strstr(aCommand->c_str(), "scc.exe") == nullptr)
+    {
+        return Global_ExecuteProcess(a1, aCommand, aArgs, aCurrentDirectory, a5);
+    }
+    std::wstring original = aArgs->str;
+    std::wstring buffer;
+    auto scriptSystem = App::Get()->GetScriptSystem();
+    if (scriptSystem->IsUsingRedmod())
+    {
+        spdlog::info("Using RedMod configuration");
+        buffer = scriptSystem->GetRedModArgs();
+    }
+    else
+    {
+        buffer = aArgs->str;
+    }
+
+    buffer += LR"( -compilePathsFile ")" + scriptSystem->CreatePathsFile() + L'"';
```

Comment:
So this flag is telling the game or `scc.exe` to look for that file and read all the paths?

Reply:
Yeah, I'm working on the needed changes to scc here: https://github.com/jackhumbert/redscript/tree/feature/compile_paths_file